lib/beyonic/collection.rb

```diff
@@ -2,4 +2,10 @@
 class Beyonic::Collection < OpenStruct
   include Beyonic::AbstractApi
   set_endpoint_resource "collections"
-end
\ No newline at end of file
+
+  def self.claim(amount, payload={})
```

Comment:
So I should change it to
```
self.claim(amount, phonenumber, remote_transaction_id)
```
?

BTW staging api seems ignored this parameters, and the only collection present at my account (id: 1, organization: 5) has remote transaction id equal to null, is it normal?